posta/models.py

```diff
@@ -150,7 +150,9 @@ def invia(self, connection=None):
 
         # E-mail a delle persone
         for d in self.oggetti_destinatario.filter(inviato=False):
-
+            destinatari = [d.persona.email]
+            if utenza and d.persona.utenza.email != d.persona.email:
```

Comment:
Seguire la relazione persona.utenza potrebbe lanciare una eccezione, nel caso di utenza non esistente. Solo meta' delle persone su Gaia ha una utenza (l'altra meta' e' stata inserita d'ufficio). 

Il comportamento desiderato e' ignorare il flag `utenza` nel caso in cui la persona non abbia una utenza (e mandare solo alla e-mail di contatto, se esistente).